torch/csrc/distributed/c10d/SymmetricMemory.cpp

```diff
@@ -276,6 +276,8 @@ TORCH_LIBRARY_FRAGMENT(symm_mem, m) {
       "memset32_(Tensor(a!) input, int offset, int val, int count) -> Tensor(a!)");
 
   m.def("nvshmem_broadcast(Tensor(a!) input, str group_name) -> Tensor(a!)");
+  m.def(
```

Comment:
For my learning, this is pybind registration right?

Reply:
Yeah, similar. A method would be added to `torch.ops.symm_mem`.